x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/getting_started_steps/icon_row.scss

```diff
@@ -0,0 +1,11 @@
+.grayscaleSvg {
+    filter: grayscale(1);
+}
+
+.addManyMoreButton {
+  border: $euiBorderThin;
+}
+
+.iconTooltip {
+  border-bottom: $euiBorderThin;
+}
```

Comment:
I'm a little concerned with the generic CSS class names. There's at least a remote possibility that other plugins could have items with the same name. Perhaps you could namespace this under a parent `.gettingStartedIconRow` selector or something like that. 

Something like:

```css
.gettingStartedIconRow {

  .grayscaleSvg {
      filter: grayscale(1);
  }

  .addManyMoreButton {
    border: $euiBorderThin;
  }

  .iconTooltip {
    border-bottom: $euiBorderThin;
  }

}
```
Then:

```JavaScript
 <EuiFlexGroup gutterSize="s" alignItems="center" className="gettingStartedIconRow">
...
```